README.md

```diff
@@ -0,0 +1,15 @@
+# fec-infrastructure
+
+Terraform configuration and Concourse pipeline for managing custom FEC infrastructure in AWS.
+
+## Setup
+* Concourse pipeline is deployed to a concourse team owned by 18F Infrastructure
+* Terraform configuration is deployed via Concourse to an AWS account owned by Infrastructure
+* FEC staff are responsible for adding resources that incur a cost to Chandika
```

Comment:
Two parts to this:
* FEC should tag every resource with the appropriate tag (whatever that is!) via terraform, and infrastructure should check those tags when reviewing pull requests.
* FEC should list resources in chandika so that chandika doesn't destroy them.